### PR TITLE
feat: implement `MigrationReviewStep`

### PR DESCRIPTION
**Pull request by goga-m**
## Summary

Closes https://app.clickup.com/t/39xfaxy

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [x] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Missing background on this icon

Design
image

Imp
image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Button should read "Back" instead of "Cancel"

Design
image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Should be 12px between "Amount Migrated" and the value. Looks a bit less than that atm.

Design
image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
This icon should be `gray-200` in dark mode

image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Completed icon missing background.

Design
image

Imp
image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
640 & 375. The content is too wide. Should be 32px of padding around the border of content.

The border is currently within the padding as shown here:
image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
375 - May need more truncation of the wallet address. Icon currently overlaps the text.

Des
image

Imp
image

File: src/domains/migration/components/MigrationReviewStep/MigrationReviewStep.blocks.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { Icon } from "@/app/components/Icon";
+
+export const MigrationPolygonIcon = () => (
+	<>
+		<div className="absolute top-0 right-8 bottom-0 flex w-8 items-center justify-center bg-white text-white text-theme-secondary-300 dark:bg-theme-secondary-900 dark:text-theme-secondary-800">
+			<Icon name="Polygon" size="xl" />
+		</div>
+
+		<div className="text-purple-50 absolute top-0 right-8 bottom-0 flex w-8 items-center justify-center">
+			<Icon name="DoubleArrowDown" size="lg" />
```
Comment (dated) [inline src/domains/migration/components/MigrationReviewStep/MigrationReviewStep.blocks.tsx:11]: I think we already have that icon as `Received`

**Comment by dated** [comment]
> Completed icon missing background.
> 
> Design image
> 
> Imp image

fixed in https://github.com/ArdentHQ/arkvault/pull/282

**Comment by goga-m** [comment]
@samharperpittam @dated fixed the issues.

**Comment by ItsANameToo** [comment]
conflicts @goga-m

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**